Dockerfile.dapper401475757

```diff
@@ -0,0 +1,53 @@
+ARG GOLANG=golang:1.15.15-alpine3.13
```

Comment:
Don't think this file should be included in the PR

Reply:
upss!! thanks!